src/caselawclient/Client.py

```diff
@@ -42,23 +42,31 @@
 DEFAULT_XSL_TRANSFORM = "accessible-html.xsl"
 
 
-def decode_multipart(response: requests.Response) -> str:
-    """Decode a multipart response and return just the text inside it.
-    Note that it is possible for multiple responses to be returned, if
-    multiple top-level returns exist in the XQuery."""
+class MultipartResponseLongerThanExpected(Exception):
+    pass
 
-    # Arguably, this should return None -- it occurs when there are no
-    # matching entries
+
+def get_multipart_strings_from_marklogic_response(
+    response: requests.Response,
+) -> list[str]:
+    # TODO: This is horrible, and should either return a None, or throw an exception.
     if not (response.content):
-        return ""
+        return [""]
```

Comment:
Is it worth having a pure function which returns None and a wrapper which transforms it into `[""]`, so future code can be better?

Reply:
I'm definitely not happy with this as it stands, and would prefer returning a `None`. We could always make the `get_string` flavour return `""` in case of an empty response, just to reduce downstream breakage.